src/peer/connection.js

```diff
@@ -126,6 +137,8 @@ class Connection extends EventEmitter {
   _handleQueuedMessages() {
     for (const message of this._queuedMessages) {
       switch (message.type) {
+        // Should we remove this ANSWER block
+        // because ANSWER should be handled immediately?
```

Comment:
I agree with you.
I suppose that all received messages should be handled immediately except for ice candidates received before the peer connection have a remote description.